apps/wing-console/console/ui/src/layout/status-bar.tsx

```diff
@@ -29,18 +33,31 @@ export const StatusBar = ({
     success: "success",
     error: "error",
   };
+
+  const resetMutation = trpc["app.reset"].useMutation();
+  const [showRestartModal, setShowRestartModal] = useState(false);
+
+  const restart = useCallback(async () => {
+    setShowRestartModal(false);
+    await resetMutation.mutateAsync();
+  }, [resetMutation]);
+
   return (
     <footer
       className={classNames(
-        theme.bg3,
+        theme.bg1,
         theme.text1,
         theme.border3,
-        "py-1 px-4 flex text-2xs w-full relative z-10",
+        "px-4 flex items-center text-2xs w-full relative z-10",
       )}
     >
       {/*left side*/}
-      <div className="w-full flex space-x-6">
-        <div title={wingVersion} className="truncate space-x-1 min-w-[7rem]">
+      <div className="w-full flex gap-4 items-center">
```

Comment:
```suggestion
      <div className="flex gap-2 items-center">
```

No need to grow. Let's standardize the gap to 2 (in sync with the rest of the status bar.